examples/graphql-client-handle-partial-response/graphql_client_handle_partial_response.bal

```diff
@@ -9,12 +9,17 @@ import ballerina/io;
 // the meta information on protocol extensions from the GraphQL server.
 type ProfileResponse record {|
     *graphql:GenericResponseWithErrors;
-    record {|Profile profile?;|} data;
+    record {|Profile profile;|} data;
 |};
 
+// If `Field Errors` are raised from the GraphQL server side for the `age` and `name` fields then
+// the server sets `null` values for these fields. If these fields are of `NON_NULL` type then the
+// errors are further propagated to their parent fields. The following record type defines the
+// shape of the response from a GraphQL service which allows the `name` and `age` fields to have a
+// `null` value.
```

Comment:
I think the part of the description should be in the BBE description, rather than in the code comment. 

This comment is not clear enough. We should be talking about `non null` fields and how to handle them. Not the other way around. Error propagating is not related here.

Reply:
Fixed in [#6f34b2d](https://github.com/ballerina-platform/ballerina-distribution/pull/3731/commits/6f34b2d68817e114687f109a55bfa5345aa50afd)